flocker/control/schema/types.yml

```diff
@@ -175,6 +175,14 @@ definitions:
       and optional tag, e.g. "clusterhq/redis:latest".
     type: string
 
+  command_line:
+    title: "Command line"
+    description: |
+      Alternate command-line to run using the Docker image's entry point.
+    type: array
+    items:
+      type: string
```

Comment:
Limit on the length of the array and length of the strings in the array makes sense here, I think (though overall maximum length would be really nice, I have no idea how to express that in jsonschema).

Some grepping suggests 4096 * 32 is the overall maximum on Linux.